### DISCUSSION TITLE
AWS Kubernetes Support + Command Menu

### DISCUSSION THREAD

**Discussion by marcuskohlberg** [encoredev/encore#929] [General]
## Amazon EKS Support
We're happy to share that Encore now supports deploying to Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS)! 🎉
_This also means you can now deploy your app to Kubernetes on both GCP and AWS._

Encore uses EKS AWS Fargate (which is a very confusing name — Thanks Amazon!) to run your Kubernetes applications using serverless compute. This removes the need to provision and manage servers, lets you specify and pay for resources per application, and improves security through application isolation by design.

To deploy your app to EKS: Go to **Create Environment**, select **Amazon Web Services** as the cloud, then select **Kubernetes: EKS Fargate**.

![aws eks](https://github.com/encoredev/encore/assets/78424526/23b809f3-db38-4797-a94b-783519d9de77)

## Command Menu
As we add more and more features to Encore, we also want to make navigation simpler and faster. We're now introducing the **Command Menu** as a new lightning-fast method of moving around the Cloud Dashboard (coming soon to Local Dev Dash!).⚡️

Try it out yourself by opening your app in the [Cloud Dashboard](https://app.encore.dev) and pressing **⌘+K** (Mac) or **Ctrl+K**!

In this first release, Command Menu lets you search for pages in the Cloud Dashboard and Docs. Soon we'll add much more functionality, like the ability to search for apps, services, endpoints, and more.

We'd love to hear your feedback and ideas for how to improve it — jump on [Slack](https://encore.dev/slack) and share your thoughts!

https://github.com/encoredev/encore/assets/78424526/2504ea8d-b83f-42de-bd93-d97a763cb525

## Thanks to all contributors
🙏 We continue to be overwhelmed by your support, feedback, and suggestions!
Together we're building the future of backend development and we couldn't be more excited.

❤️ As always, we're excited to hear what you think!
Please share your feedback on [Slack](https://encore.dev/slack).

**Full Changelog**: https://github.com/encoredev/encore/compare/v1.27.7...v1.28.0

<hr /><em>This discussion was created from the release <a href='https://github.com/encoredev/encore/releases/tag/v1.28.0'>AWS Kubernetes Support + Command Menu</a>.</em>

**Comment by mdwitr0** [comment]
Hi, is there any plans to support a personal kubernetes cluster without AWS?

**Reply by marcuskohlberg** [reply]
Hey, great question! Not in the short term.
For small scale use, you can use Encore's built-in development cloud for free (fair use limits) — if cost is your concern. Or if you have a use case where you have to run it yourself, you can potentially use the `encore eject` command to create a standalone docker image and deploy that. ([See docs](https://encore.dev/docs/how-to/migrate-away#ejecting-your-app-as-a-docker-image))
